cardinal/server/server.go

```diff
@@ -58,14 +52,16 @@ func New(engine *ecs.Engine, wsEventHandler func(conn *websocket.Conn), opts ...
 	setupRoutes(app, engine, wsEventHandler, s.config)
 
 	if !s.config.isSwaggerDisabled {
-		if err := setupSwagger(app); err != nil {
-			return nil, err
-		}
+		setupSwaggerRoutes(app)
 	}
 
 	return s, nil
 }
 
+func setupSwaggerRoutes(app *fiber.App) {
+	app.Get("/swagger/*", swagger.HandlerDefault)
```

Comment:
let's consolidate this into `setupRoutes` it already have access to `s.Config` anyway, so there's no extra dependency injection needed.

This is preferable because it consolidates all part of the code that has to do with registering routes into a single function making it easier to trace.